pkg/inferenceservice-controller/testdata/crd/serving.kserve.io_inferenceservices.yaml

```diff
@@ -0,0 +1,16330 @@
+---
```

Comment:
question: instead of having this file locally, can we use from kserve repo like:
```
https://github.com/kserve/kserve/blob/release-0.13/charts/kserve-crd/templates/serving.kserve.io_inferenceservices.yaml
```

by indicating a link of sort to envtest?